airflow/decorators/base.py

```diff
@@ -165,7 +166,7 @@ def __init__(
         python_callable: Callable,
         task_id: str,
         op_args: Optional[Collection[Any]] = None,
-        op_kwargs: Optional[Mapping[str, Any]] = None,
+        op_kwargs: Optional[MutableMapping[str, Any]] = None,
```

Comment:
I don’t think this needs to be mutable?

Reply:
Right. Fixed by copying ok_kwargs before mutating it .